features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Resources/components/APIRateLimiting.jsx

```diff
@@ -63,7 +63,11 @@ function APIRateLimiting(props) {
     // control this component, Then we accept the props as the valid input and update the current state value from props
     useEffect(() => {
         if (onChange) {
-            setApiThrottlingPolicy(currentApiThrottlingPolicy);
+            if (currentApiThrottlingPolicy === '' && apiFromContext.apiThrottlingPolicy) {
```

Comment:
I think `currentApiThrottlingPolicy` should not have an `` empty string at any time.

Reply:
Once you select Operation Level and then select API Level currentApiThrottlingPolicy becomes an `` empty string